src/Jackalope/ObjectManager.php

```diff
@@ -869,6 +869,7 @@ public function save()
                 if ($item->isModified() || $item->isMoved()) {
                     $item->confirmSaved();
                 }
+                $item->setClean();
```

Comment:
@dantleech can we do the solution i propose in my last comment, or does that lead to some other problem?